timelock-api/src/main/conjure/timelock-management-api.yml

```diff
@@ -59,3 +59,21 @@ services:
           external intervention to start up the service again. The endpoint returns the (randomly generated) server id
           associated with server upon start up. A client can then make `getServerId` requests wherein change in
           server id would imply that the server was bounced.
+
+      fastForwardTimestamp:
+        http: POST /fastForward
+        args:
+          namespace:
+            type: string
+            safety: safe
+            param-type: query
+          currentTimestamp:
+            type: safelong
```

Comment:
see timelock-api - this should be a Long, not a safelong. They have different max sizes :(